addons/pos_loyalty/static/tests/tours/PosLoyaltyLoyaltyProgramTour.js

```diff
@@ -15,18 +15,18 @@ registry.category("web_tour.tours").add("PosLoyaltyLoyaltyProgram1", {
             // Order1: Generates 2 points.
             ProductScreen.addOrderline("Whiteboard Pen", "2"),
             ProductScreen.clickPartnerButton(),
-            ProductScreen.clickCustomer("Test Partner AAA"),
+            ProductScreen.clickCustomer("AAA Test Partner"),
```

Comment:
Can you elaborate a little on the reason why the partners weren't shown, and why renaming them fixes the issue?

Reply:
Oh ok, that's cleaver. :smile: 
But then, shouldn't we name them `AAA`, `AAB`, `AAC` just to be sure? Idk, I trust you. :shrug: